crates/fuel-core/src/p2p_test_helpers.rs

```diff
@@ -606,18 +606,18 @@ impl Node {
         }
     }
 
-    /// Wait for the node to reach consistency with the given transactions within 10 seconds.
+    /// Wait for the node to reach consistency with the given transactions within 30 seconds.
     pub async fn consistency_10s(&mut self, txs: &HashMap<Bytes32, Transaction>) {
-        tokio::time::timeout(Duration::from_secs(10), self.consistency(txs))
+        tokio::time::timeout(Duration::from_secs(30), self.consistency(txs))
```

Comment:
are these still needed?

Reply:
replace tool didn't worked well zzz

Reply:
The test of p2P keeps failing xd while it pass on other pr

Reply:
it seems that @xgreenx committed this in purpose : https://github.com/FuelLabs/fuel-core/pull/2913/commits/c4fb3d2c1ed35e1e906028a47ac49a91ad16cee0 so I add again